### PR TITLE
Instructors page

### PR DESCRIPTION
**Pull request by drouba**
Instructor page set up
add is_instructor attribute to user
add create instructor function on instructor page
instructor page shows list of all instructors

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -9,7 +9,12 @@
   </li>
   <li>
     <a href="discussions.php"><strong>View</strong></a> - list all discussions
+  </li> 
+  <li>
+    <a href="InstructorsList.php"><strong>View</strong></a> - Instructors
   </li>
```
Comment (jamestjw) [inline public/index.php:15]: maybe we should stick to naming the files in lowercase

File: install.php [hunk 2]
```diff
@@ -29,4 +29,4 @@
 $admin->email = "admin@concordia.ca";
 $admin->is_admin = 1;
 $admin->password_digest = password_hash('root', PASSWORD_DEFAULT);
-$admin->save();
\ No newline at end of file
+$admin->save();
```
Comment (jamestjw) [inline install.php]: you'll need to add a default value for is_instructor here too

**Comment by jamestjw** [comment]
create_user.php is also not working since it doesn't have a default value for is_instructor

File: public/index.php (hunk 1 above)
Comment (drouba) [reply public/index.php:15]: done

File: install.php (hunk 2 above)
Comment (drouba) [reply install.php]: do you think admin should also be an instructor by default or not?

Comment (drouba) [reply install.php]: set it to false for now, can be changed later if we decide otherwise

**Comment by drouba** [comment]
fixed create_user.php (added a default value to is_instructor attribute)

**Review by jamestjw — APPROVED** [review]

**Pull request merged**